tests/test_cli.py

```diff
@@ -14,6 +14,18 @@ class TestCli:
     def initdir(self, tmpdir):
         self.path = (tmpdir / "data").mkdir()
 
+    # TODO extend with test data for 0.2
+    @pytest.fixture(params=["0.1", "0.3"], ids=["v0.1", "v0.3"])
+    def s3_address(self, request):
+        urls = {
+            "0.1": "https://s3.embassy.ebi.ac.uk/idr/zarr/v0.1/6001240.zarr",
+            "0.3": (
+                "https://minio-dev.openmicroscopy.org/idr/v0.3/"
```

Comment:
@will-moore : are you ready for us to move some of the data to the IDR bucket?

Reply:
Migrated the flower to idr/zarr/v0.3:
![Screen Shot 2021-08-25 at 15 26 39](https://user-images.githubusercontent.com/88113/130798979-68aed3d6-e839-4b7c-9525-bf3435773cc0.png)